### PR TITLE
HMS-4406: new policy api's

### PR DESCRIPTION
**Pull request by amit-100ms**
Need to update from qa to prod
Need to explain the parameters a bit more.

### CODE REVIEW

**Comment by amit-100ms** [comment]
what else is required? added correct validations - showing 2 commits of others in this PR.

File: docs/server-side/v2/foundation/templates-and-roles.mdx [hunk 1]
```diff
@@ -4,3 +4,650 @@ nav: 2.2
...
+| bitRate | `int`    | Audio Max bitrate of audio track in kbps.<br/>Range: `16`-`128`<br/>Default: `32` | No       |
+| codec   | `string` | Codec for the audio track.<br/>Options: [`opus`] <br/>Default: `opus`             | No       |
+
+#### video
+
+| Name      | Type     | Description                                                                   | Required |
+| :-------- | :------- | :---------------------------------------------------------------------------- | :------- |
+| bitRate   | `int`    | Max bitrate of video track in kbps.<br/>Range: `50`-`2000`<br/>Default: `256` | No       |
+| codec     | `string` | Codec for the video track.<br/>Options: [`vp8`]<br/>Default: `vp8`            | No       |
+| FrameRate | `int`    | Max number of video frames per second.<br/>Range: `1`-`30`<br/>Default: `25`  | No       |
```
Comment (sagargaonkar) [inline docs/server-side/v2/foundation/templates-and-roles.mdx:181]: frameRate

File: docs/server-side/v2/foundation/templates-and-roles.mdx [hunk 2]
```diff
@@ -4,3 +4,650 @@ nav: 2.2
...
+  {
+    "rid": "q",
+    "scaleResolutionDownBy": 4.0,
+    "maxBitrate": 1000,
+    "maxFramerate": 10
+  }
+]
+```
+
+#### subcribeParams
```
Comment (sagargaonkar) [inline docs/server-side/v2/foundation/templates-and-roles.mdx:256]: subscribeParams

File: docs/server-side/v2/foundation/templates-and-roles.mdx [hunk 3]
```diff
@@ -4,3 +4,650 @@ nav: 2.2
 ---
 
 <Content alias="templatesAndRoles" />
+
+## API reference
+
+Apart from the dashboard, a programmatic way to interact with templates is via API. You will need a [management token](authentication-and-tokens) as a bearer token for your requests. After updating a template, you might need to rejoin or initiate a new session for the template updates to take place.
+
+### Common Headers For all APIs
```
Comment (sagargaonkar) [inline docs/server-side/v2/foundation/templates-and-roles.mdx:12]: for

**Comment by amit-100ms** [comment]
Removed simulcast.

**Review by sagargaonkar — APPROVED** [review]

**Pull request merged**